pandas/sparse/tests/test_sparse.py

```diff
@@ -742,6 +743,60 @@ def test_combine_first(self):
         assert_sp_series_equal(result, result2)
         assert_sp_series_equal(result, expected)
 
+    def test_to_coo(self):
```

Comment:
create a new class to test with. then you can do the setup in ``setUp`` (just inherit from ``TestCase``) and will work